VSDiagnostics/VSDiagnostics/VSDiagnostics/Utilities/Extensions.cs

```diff
@@ -131,5 +132,12 @@ public static bool IsAlias(this string type)
 
             return _aliasMapping.ContainsKey(type);
         }
+
+        public static bool IsAsync(this IMethodSymbol methodSymbol)
```

Comment:
Could you add some XML documentation to it so we see it pop up in-editor? It's definitely going to cause confusion but I can't immediately come up with a better name so we should at least clarify it through that.